lib/bolt/inventory.rb

```diff
@@ -108,6 +109,15 @@ def vars(target)
       @target_vars[target.name]
     end
 
+    def add_facts(target, new_facts = {})
+      @logger.warn("No facts to add")
```

Comment:
This warning should only happen if `new_facts` is empty.

Reply:
Ack, good catch!